perun-web-gui/src/main/java/cz/metacentrum/perun/webgui/json/tasksManager/GetFacilityState.java

```diff
@@ -187,7 +188,10 @@ else if (row.getState().equalsIgnoreCase("OPEN")){
 				else if (row.getState().equalsIgnoreCase("ERROR")){
 					return "rowred";
 				}
-		return "";
+				else if (row.getState().equalsIgnoreCase("WARNING")){
```

Comment:
``FacilityState`` is object of its own with states ``OK, ERROR, PROCESSING, NOT_DETERMINED`` so please remove ``OPEN`` in above "else if" and use colors green, red, yellow, white accordingly. Also ``WARNING`` shouldn't be here.